src/Modules/Build.lua

```diff
@@ -156,11 +156,57 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		end
 	end
 	self.controls.pointDisplay.width = function(control)
-		local used, ascUsed = self.spec:CountAllocNodes()
-		local usedMax = 99 + 22 + (self.calcsTab.mainOutput.ExtraPoints or 0)
-		local ascMax = 8
-		control.str = string.format("%s%3d / %3d   %s%d / %d", used > usedMax and "^1" or "^7", used, usedMax, ascUsed > ascMax and "^1" or "^7", ascUsed, ascMax)
-		control.req = "Required level: "..m_max(1, (100 + used - usedMax))
+		if (self.calcsTab.mainOutput.ExtraPoints == nil) 
+		then 
+		  bandit = 0 
+		else 
+		  bandit = self.calcsTab.mainOutput.ExtraPoints or 0 
+		end
+		
+		local PointsUsed, AscUsed = self.spec:CountAllocNodes()
+		local banditStr = ""
+		local usedMax, ascMax, levelreq = 99 + 22 + bandit, 8, 1
+		
+		-- level per act as of ingame description last waypoint, 10 entries
+		local act = {12,22,32,40,44,50,54,60,64,67}
+		-- skillpoints accumulated without bandits per act, 10 entries
+		local skillpoints = {2,3,5,6,8,11,14,17,19,22}
+				
+		-- when I'm already "fixing" the leveling info, I'm adding a suggestion for running the lab as well :)
+		local labSuggest, labstr = "", {"\nLabyrinth: Normal Lab", "\nLabyrinth: Cruel Lab", "\nLabyrinth: Merciless Lab", "\nLabyrinth: Uber Lab"}
+		
+		-- cursor
+		local cursor = 1
+						
+		-- loop for how much quest skillpoints are used with the progress
+		repeat
+		  if PointsUsed >= act[cursor]+skillpoints[cursor]
+		    then 
+			  cursor=cursor+1 
+		  else 
+			break 
+		  end
+		until cursor>=10
+
+		-- bandits notification should only be available when considered and after act 2
+		if (cursor >=2 and bandit ~= 0) then
+			banditStr = "\nBandits Skillpoints: " .. bandit
+		end
+		
+		-- to prevent a negative level at a blank sheet the level requirement will be set dependent on points invested until catched up with quest skillpoints 
+		levelreq = math.max(PointsUsed - skillpoints[cursor],1)
+		if (levelreq <= skillpoints[cursor]) then levelreq = PointsUsed+1 end  
+		
+		-- Ascendency points for lab
+		-- this is a recommendation for beginners who are using Path of Building for the first time and trying to map out progress in PoB
+		if levelreq >= 33 and levelreq <= 38 then labSuggest= labstr[1] end
+		if levelreq >= 55 and levelreq <= 60 then labSuggest= labstr[2] end
+		if levelreq >= 68 and levelreq <= 73 then labSuggest= labstr[3] end
+		if levelreq >= 75 and levelreq <= 85 then labSuggest= labstr[4] end -- a little bit more leeway for Uber Lab
```

Comment:
These level intervals seem arbitrary, and should probably just be minimum level up to the minimum level of the next labyrinth.  Also, I'd define the labstr closer to where it's being used.